src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -1342,6 +1342,21 @@ namespace DataHeatBalance {
 		//   less out of bounds temperature errors allowed
 		int ZoneMinCO2SchedIndex; // Index for the schedule the schedule which determines minimum CO2 concentration
 		int ZoneContamControllerSchedIndex; // Index for this schedule
+		bool FlagCustomizedZoneCap; // True if customized Zone Capacitance Multiplier is used
+		// Hybrid Modeling
+		bool FlagHMInternalThermalMass; // True if hybrid modeling (Internal thermal mass calculations) is used for the Zone
+		bool FlagHMInfiltration;  // True if hybrid modeling (Infiltration calculations) is used for the Zone
```

Comment:
These two flags don't belong here - the globals are in HybridModel if I'm following correctly.  Delete these.